### PR TITLE
2.5.0 release

### PR DESCRIPTION
**Pull request by DavertMik**

### CODE REVIEW

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/Playwright.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/Playwright.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+* **Experimental: [Playwright](/playwright) helper introduced**.
+
+> [Playwright](https://github.com/microsoft/playwright/) is an alternative to Puppeteer which works very similarly to it but adds cross-browser support with Firefox and Webkit. Until v1.0 Playwright API is not stable but we introduce it to CodeceptJS so you could try it.
+
+* [Puppeteer] Fixed basic auth support when running in multiple sessions. See #2178 by @ian-bartholomew
+* [Puppeteer] Fixed `waitForText` when there is no `body` element on page (redirect). See #2181 by @Vorobeyko
+* [Selenoid plugin] Fixed overriding current capabilities by adding deepMerge. Fixes #2183 by @koushikmohan1996
+* Added types for `Scenario.todo` by @Vorobeyko
+* Added `--colors` option to `run` and `run-multiple` so you force colored output in dockerized environment. See #2189 by @mirao
+
```
Comment (PeterNgTr) [inline CHANGELOG.md]: Maybe we miss those #2166, #2164, #2169

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ lib/helper/Playwright.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

**Comment by PeterNgTr** [comment]
```
- HELPER=Playwright 
```

should we remove Playwright tests in travis? It is triggered by github action already.

**Comment by DavertMik** [comment]
> should we remove Playwright tests in travis? It is triggered by github action already.

No, @PeterNgTr they are different. 

* unit tests (helpers/**.js) on Travis
* e2e tests (tests/acceptance/**) on Circle CI for other helpers and on Github Actions for Playwright

File: test/helper/Playwright_test.js [hunk 2]
```diff
@@ -71,8 +71,11 @@ describe('Playwright', function () {
       url: siteUrl,
       windowSize: '500x700',
       show: false,
+      browser: 'firefox',
+      show: true,
```
Comment (DavertMik) [inline test/helper/Playwright_test.js]: Codacy Issue found: [Duplicate key 'show'.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones removed
==============
+ lib/helper/Playwright.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

File: CHANGELOG.md (hunk 1 above)
Comment (DavertMik) [reply CHANGELOG.md]: No they were referenced in previous release

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ lib/helper/Playwright.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

File: test/helper/Playwright_test.js [hunk 3]
```diff
@@ -71,6 +71,8 @@ describe('Playwright', function () {
...
```
Comment (PeterNgTr) [inline test/helper/Playwright_test.js]: duplicate `show` key

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ lib/helper/Playwright.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

File: docs/helpers/Playwright.md [hunk 4]
```diff
@@ -19,7 +19,9 @@ Uses [Playwright][1] library to run tests inside:
 
 This helper works with a browser out of the box with no additional tools required to install.
 
-Requires `playwright` package to be installed.
+Requires `playwright` package version ^0.10.0 to be installed:
+
+    npm i playwright@^0.10.0 --save
```
Comment (PeterNgTr) [inline docs/helpers/Playwright.md]: I think we should update the version here. `^0.11.0`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,22 @@
+## 2.5.0
+
+* **Experimental: [Playwright](/playwright) helper introduced**.
+
+> [Playwright](https://github.com/microsoft/playwright/) is an alternative to Puppeteer which works very similarly to it but adds cross-browser support with Firefox and Webkit. Until v1.0 Playwright API is not stable but we introduce it to CodeceptJS so you could try it.
```
Comment (PeterNgTr) [inline CHANGELOG.md]: is an alternative to Puppeteer which works very similarly to it but adding cross-browser support with Firefox and Webkit that brings Playwright to the next level. Until Playwright v1.0 is released, current APIs are not stable but we'd like to  introduce it to CodeceptJS so you could have chance to try it out.

File: CHANGELOG.md (hunk 1 above)
Comment (PeterNgTr) [reply CHANGELOG.md]: sorry, I misread the release notes :)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/Playwright.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

File: docs/helpers/Playwright.md (hunk 4 above)
Comment (DavertMik) [reply docs/helpers/Playwright.md]: Agreed

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/Playwright.js  1
         

Clones removed
==============
+ test/helper/Playwright_test.js  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/Playwright.js  1
         

Clones removed
==============
+ test/helper/Playwright_test.js  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/Playwright.js  1
         

Clones removed
==============
+ test/helper/Playwright_test.js  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/Playwright.js  1
         

Clones removed
==============
+ test/helper/Playwright_test.js  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/Playwright.js  1
         

Clones removed
==============
+ test/helper/Playwright_test.js  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4987580&bid=16467426)

**Pull request merged**